### ISSUE TITLE
Understanding Nucleoatac outputs and Figure 7a from paper

### ISSUE THREAD

**Issue by Flope** [GreenleafLab/NucleoATAC#83]
I am working on my first ATAC-seq project and I am trying to go through all the nucleoatac outputs.
I have read the manuscript and the docs on the website. 
I guess I can summarize my questions as 
**what data did you use to plot figure 7A from the outputs? (what are the # of calls)?**
![image](https://user-images.githubusercontent.com/3597418/47813632-09551e80-dd22-11e8-8cbb-147d0aab35bd.png)
The data was prefiltered. 

> For all analyses, only calls with Z-scores >3 (corresponding to a P-value of ∼0.001) and log-likelihood ratios >0 were considered.

It makes me think that I should be using output_basename.nucpos.bed.gz Correct?
and I should use the same output to plot the occupancy scores (column 4)  for figure 7B ? 

output_basename.nucpos.bed.gz is 1bp resolution bed file, right? In here dyad position means position within the nucleosome. I am a bit confused about the term dyad and how it is used in here. 

For the motifs, did you use the midpoint of the motif?

Thank you very much for your time and help

**Comment by AliciaSchep** [comment]
Yes used *.nucpos.bed.gz.  Dyad position here means the center point of the DNA that would be wrapped around the nucleosome.  For motifs, yes used the midpoint of motif.
